BlueprintUICommonControls/Sources/Box.swift

```diff
@@ -146,9 +146,10 @@ extension Box.CornerStyle {
         case .square:
             return 0
         case .capsule:
-            return min(bounds.height, bounds.width) / 2.0
+            return min(bounds.midX, bounds.midY)
```

Comment:
Fwiw, while this is less code; its also a bit only based on the fact that bounds is always at 0,0. If it was not at 0,0, this would be reporting the middle of the frame in its coordinate space, whereas the `/ 2` on the width and height makes this explict.

Reply:
Ah that makes sense.. just to make sure I understand, a rect of `x: 10, y: 10, width: 20, height: 20` would have a `midX` of 15, right? Will go back to `/ 2` 👍

Reply:
Yeah, it would